app/management/commands/notify_keywords.py

```diff
@@ -0,0 +1,187 @@
+from django.core.management.base import BaseCommand
```

Comment:
Maybe create a file with common code used by notify fav and notify keywords so just a little code is duplicated.